### ISSUE TITLE
Creating tabs dynamically?

### ISSUE THREAD

**Issue by securitybryan** [bahadirsofuoglu/form-wizard-vue3#37]
Is it possible to dynamically create tabs?  I have a wizard in Backbone I am moving to Vue 3 and need to create tabs on the fly depending upon the path the user selects.

**Comment by g1ronn1mo** [comment]
No, this is not possible on the native way. I tried it, and the form gets preloaded in the first place. Every TabComponent that is dynamically loaded gets pushed to the end.

**Comment by securitybryan** [comment]
Thank you for letting me know.   It is surprising how difficult it seems to be to get simple tabs working with Vue 3.
